<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Options</class>
 <widget class="QDialog" name="Dialog_Options">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diamond Options</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <item row="8" column="1">
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Main Dictionary (dic):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Time Format:</string>
     </property>
    </widget>
   </item>
   <item row="25" column="3">
    <widget class="KeyLineEdit" name="key_spellCheck">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="17" column="3">
    <widget class="KeyLineEdit" name="key_upper">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="26" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="16" column="1">
    <widget class="QLabel" name="label_12">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edit - Select Block:</string>
     </property>
    </widget>
   </item>
   <item row="24" column="1">
    <widget class="QLabel" name="label_14">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tools - Play Macro:</string>
     </property>
    </widget>
   </item>
   <item row="28" column="0" colspan="7">
    <layout class="QHBoxLayout" name="horizontalLayout_101">
     <item>
      <spacer name="horizontalSpacer_100">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save_PB">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_101">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_PB">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_103">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="25" column="1">
    <widget class="QLabel" name="label_15">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tools - Spell Check:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3" colspan="3">
    <widget class="QLineEdit" name="dictUser">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="statusTip">
      <string extracomment="Location of User Dictionary"/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="tabSpacing_CB">
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="dateFormat_CB">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edit - Select Line:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>User Dictionary (txt):</string>
     </property>
    </widget>
   </item>
   <item row="14" column="3">
    <widget class="KeyLineEdit" name="key_selectWord">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="13" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="22" column="3">
    <widget class="KeyLineEdit" name="key_columnMode">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="11" column="6">
    <widget class="QToolButton" name="about_TB">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="13" column="3">
    <widget class="KeyLineEdit" name="key_selectLine">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="19" column="1">
    <widget class="QLabel" name="label_16">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edit - Indent Increase:</string>
     </property>
    </widget>
   </item>
   <item row="20" column="1">
    <widget class="QLabel" name="label_17">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edit - Indent Decrease:</string>
     </property>
    </widget>
   </item>
   <item row="19" column="3">
    <widget class="KeyLineEdit" name="key_indentIncr">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="23" column="3">
    <widget class="KeyLineEdit" name="key_goLine">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QLabel" name="label_11">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edit - Lowercase:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Date Format:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="6">
    <widget class="QToolButton" name="dictUser_TB">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="18" column="3">
    <widget class="KeyLineEdit" name="key_lower">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="23" column="1">
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Search - Go to Line:</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QLabel" name="label_13">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edit - Uppercase:</string>
     </property>
    </widget>
   </item>
   <item row="22" column="1">
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edit - Column Mode:</string>
     </property>
    </widget>
   </item>
   <item row="20" column="3">
    <widget class="KeyLineEdit" name="key_indentDecr">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="11" column="3" colspan="3">
    <widget class="QLineEdit" name="about">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="statusTip">
      <string extracomment="Location of about Index.html "/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edit - Select Word:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>125</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>Diamond Help:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tab Spacing:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="6">
    <widget class="QToolButton" name="dictMain_TB">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="24" column="3">
    <widget class="KeyLineEdit" name="key_macroPlay">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QCheckBox" name="useSpaces_CKB">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Use Spaces for Tabs</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3" colspan="3">
    <widget class="QLineEdit" name="dictMain">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string extracomment="Location of Main Dictionary"/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="16" column="3">
    <widget class="KeyLineEdit" name="key_selectBlock">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="timeFormat_CB">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="21" column="1">
    <widget class="QLabel" name="label_18">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edit - Delete to EOL:</string>
     </property>
    </widget>
   </item>
   <item row="21" column="3">
    <widget class="KeyLineEdit" name="key_deleteEOL"/>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="label_19">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Syntax Files:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="6">
    <widget class="QToolButton" name="syntax_TB">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3" colspan="3">
    <widget class="QLineEdit" name="syntax">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KeyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>keylineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dateFormat_CB</tabstop>
  <tabstop>timeFormat_CB</tabstop>
  <tabstop>tabSpacing_CB</tabstop>
  <tabstop>useSpaces_CKB</tabstop>
  <tabstop>dictMain</tabstop>
  <tabstop>dictMain_TB</tabstop>
  <tabstop>dictUser</tabstop>
  <tabstop>dictUser_TB</tabstop>
  <tabstop>syntax</tabstop>
  <tabstop>syntax_TB</tabstop>
  <tabstop>about</tabstop>
  <tabstop>about_TB</tabstop>
  <tabstop>key_selectLine</tabstop>
  <tabstop>key_selectWord</tabstop>
  <tabstop>key_selectBlock</tabstop>
  <tabstop>key_upper</tabstop>
  <tabstop>key_lower</tabstop>
  <tabstop>key_indentIncr</tabstop>
  <tabstop>key_indentDecr</tabstop>
  <tabstop>key_deleteEOL</tabstop>
  <tabstop>key_columnMode</tabstop>
  <tabstop>key_goLine</tabstop>
  <tabstop>key_macroPlay</tabstop>
  <tabstop>key_spellCheck</tabstop>
  <tabstop>save_PB</tabstop>
  <tabstop>cancel_PB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
